packages/smooth_app/lib/pages/product/simple_input_text_field.dart

```diff
@@ -31,90 +31,92 @@ class SimpleInputTextField extends StatelessWidget {
   final String? Function()? shapeProvider;
 
   @override
-  Widget build(BuildContext context) => Padding(
-        padding: const EdgeInsets.only(left: LARGE_SPACE),
-        child: Row(
-          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-          crossAxisAlignment: CrossAxisAlignment.center,
-          mainAxisSize: MainAxisSize.max,
-          children: <Widget>[
-            Expanded(
-              child: RawAutocomplete<String>(
-                key: autocompleteKey,
-                focusNode: focusNode,
-                textEditingController: controller,
-                optionsBuilder: (final TextEditingValue value) async {
-                  if (tagType == null) {
-                    return <String>[];
-                  }
+  Widget build(BuildContext context) {
+    final SuggestionManager manager = SuggestionManager(
+      tagType!,
+      language: ProductQuery.getLanguage()!,
+      country: ProductQuery.getCountry(),
+      categories: categories,
+      shape: shapeProvider?.call(),
+      user: ProductQuery.getUser(),
+      limit:
```

Comment:
In terms of readibility, could you add the comment just before the arg?
The formatter should work better that way

Reply:
> In terms of readibility, could you add the comment just before the arg? The formatter should work better that way

Fair enough.

Reply:
@g123k ping